src/components/views/sul-study-place/filtering-table/study-place-filtering-table.tsx

```diff
@@ -108,9 +108,9 @@ const StudyPlaceFilteringTable = ({items}: Props) => {
       <form onSubmit={e => e.preventDefault()} action="javascript:void(0);">
         <fieldset className="text-18">
           <legend className="sr-only">Filter places to study</legend>
-          <div className="mb-30 flex w-full flex-col flex-wrap items-center justify-center gap-15 *:w-full sm:flex-row sm:*:w-fit">
+          <div className="mb-30 flex w-full flex-col flex-wrap items-center justify-center gap-15 *:w-full sm:flex-row sm:*:w-fit sm:*:min-w-[250px]">
```

Comment:
Can it have the min width on all screen sizes?

Reply:
Good point! 😎